10/umbraco-cms/fundamentals/setup/upgrading/version-specific/minor-upgrades-for-umbraco-7.md

```diff
@@ -0,0 +1,103 @@
+---
+description: This articles provides details on how to upgrade to the next minor version when using Umbraco 7.
+---
+
+# Minor upgrades for Umbraco 7 and earlier versions
+
+Sometimes there are exceptions to these guidelines, which are listed in the **[version-specific guide](README.md)**.
+
+## Note
+
+It is necessary to run the upgrade installer on each environment of your Umbraco site. So if you want to update your staging and your live site then you need to repeat the steps below and make sure that you click through the install screens so that your upgrade is complete.
+
+## Contents
+
+In this article you will find instruction of 2 different ways of upgrading:
+
+* [Upgrade using NuGet](#upgrade-using-nuget)
+* [Upgrade manually from a Zip file](#upgrade-manually-from-a-zip-file)
+
+## Upgrade using NuGet
+
+1. Open up the **Package Console** and type: `Update-Package UmbracoCms`
+2. Choose **"No to All"** by pressing the **"L"** when prompted.
+    * If there are any specific configuration changes required for the version you are upgrading to then they will be noted in the **[version-specific guide](version-specific.md)**.
+
+Alternatively, you can use the Visual Studio **NuGet Package Manager** to upgrade:
+
+1. Open the **NuGet Package Manager** and select the **Updates** pane to get a list of available updates.
+2. Choose the package called **UmbracoCms** and select update.
+
+The upgrade will run through all the files and make sure you have the latest changes while leaving files you have updated.
+
+### Upgrades to versions lower than 7.2.0
+
+If you're not upgrading to 7.2.0 or higher then you should follow these extra instructions, if you are upgrading to 7.2.0+ then you can skip this and go to [Merge UI.xml and language](#merge-ui-xml-and-language).
+
+**The following only applies to upgrading to versions lower than 7.2.0**
+You will be asked to overwrite your web.config file and the files in /config, make sure to answer **No** to those questions.
+
+For some inexplicable reason, the installation will fail if you click "No to All" (in the GUI) or answer "L" (in the package manager console) to the question: "File 'Web.config' already exists in project 'MySite'. Do you want to overwrite it?" So make sure to only answer "**No**" (in the GUI) or "**N**" (in the package manager console).
+
+![](images/nuget-overwrite-dialog.png)
+![](images/nuget-upgrade-overwrite.png)
```

Comment:
Looks like the images from the article on our has not been moved over.

Can't see them on github or gitbook, however, I do see them in the article on our:
- https://our.umbraco.com/documentation/Fundamentals/Setup/Upgrading/general#upgrades-to-versions-lower-than-720